<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background-color: rgb(16, 16, 16);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-width: 2px;
	border-style: solid;
	border-color: rgb(255, 255, 255);
}

QPushButton:Hover {
	background-color: rgb(44, 44, 44);
}

QPushButton:Pressed {
	background-color: rgb(102, 102, 102);
}

QWidget {
	background-color: rgb(0, 0, 0);
	font: 28pt &quot;MS UI Gothic&quot;;
	border-radius: 5px;
}

QLabel {
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border-width: 2px;
	border-style: solid;
	border-color: rgb(102, 102, 102);;
}</string>
  </property>
  <widget class="QPushButton" name="signButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>(-)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sixButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cosButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tanButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ansButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sevenButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalsButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sqrtButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>√</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eightButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decimalButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="squareButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>x²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nineButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sinButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ln</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBracketButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBracketButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="expButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>exp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
